argusclient/model.py

```diff
@@ -12,19 +12,19 @@
 import json
 
 try:
-    basestring  # Python 2
+    str  # Python 2
 except NameError:
-    basestring = str  # Python 3
+    str = str  # Python 3
```

Comment:
why is this assignment needed

Reply:
I see that the previous approach is the most recommended, other than using `six` package. What is the reason for this change? @mveeranala

Reply:
Hari, modified it and used `six` package to address  compatibility issues.